clients/fides-js/src/lib/tcf.ts

```diff
@@ -187,6 +199,14 @@ export const initializeTcfCmpApi = () => {
   // Initialize api with TC str, we don't yet show UI, so we use false
   // see https://github.com/InteractiveAdvertisingBureau/iabtcf-es/tree/master/modules/cmpapi#dont-show-ui--tc-string-does-not-need-an-update
   window.addEventListener("FidesInitialized", (event) => {
+    // Set default GPP string if GPP is enabled. This will get updated by the GPP extension
+    if (window.Fides.experience?.gpp_settings?.enabled) {
+      window.Fides.fides_string =
+        window.Fides.fides_string ||
+        window.Fides.cookie?.fides_string ||
+        formatFidesStringWithGpp();
+    }
+
```

Comment:
it's assumed that the fides_string will already have the `DEFAULT_GPP_STRING`? 

How about for users that have already visited a page with fides and have an existing fides_string where we haven't yet added the default GPP str from this PR?

Reply:
Great call out! I always felt a little strange to do this and I think you've nailed down why. Removed this and updated the `formatFidesStringWithGpp` method to require `cmpapi` and no longer sets a default.